components/LearningUnitCheckbox.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useCallback } from "react";
+import { Box, Checkbox } from "@mui/material";
+import { useOperationMethod } from "react-openapi-client";
+
+function LearningUnitCheckbox({ learningUnitId, isPreviouslyCompleted }) {
+  const [isCompleted, setCompletion] = useState(isPreviouslyCompleted);
+
+  const [uncompleteCheckbox] = useOperationMethod(
+    "uncompleteLearningUnitForUser"
+  );
+
+  const [completeCheckbox] = useOperationMethod("completeLearningUnit");
+
+  const handleChange = useCallback(() => {
+    if (isCompleted) {
+      uncompleteCheckbox(learningUnitId);
+      setCompletion(false);
+    } else {
+      completeCheckbox(learningUnitId);
+      setCompletion(true);
+    }
+  }, [
+    learningUnitId,
+    isCompleted,
+    completeCheckbox,
+    uncompleteCheckbox,
+    setCompletion,
+  ]);
+
+  const style = {
+    position: "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 600,
+    bgcolor: "background.paper",
+    border: "2px solid #000",
+    boxShadow: 24,
+    p: 4,
+  };
+  return (
+    <Box sx={{ ...style, width: 300 }}>
+      <h1>here it goes</h1>
```

Comment:
can't read in dark mode
<img width="360" alt="image" src="https://user-images.githubusercontent.com/73831187/193714992-deb2e3f3-4a5e-44fa-82c7-77c469a8b03f.png">

Reply:
Done